### ISSUE TITLE
Follow the setting in the paper, but I can't reproduce the results.

### ISSUE THREAD

**Issue by RobitsG** [RLHF-V/RLAIF-V#28]
I train llava-7B in 8 A100 GPUs with epoch=4, batch_size=8, but the "object hallucination metric" is (24.13, 11.73) in my result, not the same as (8.5, 4.3) in the paper. Could you please point out where I am mistaken? Thank you very much.

here is my training script:

**Comment by yiranyyu** [comment]
Thank you for your interest! Results presented in the paper are produced via 4 iteration training. As we pointed in the Figure 4, it is critical to train the model in a iterative manner to overcome the distribution shift problem.

**Comment by RobitsG** [comment]
Thank you very much for your response. I have two new questions:

1) Is the RLAIF-V dataset generated by data D₄ from the fourth iteration of model M₄?
2) Has the iterative code been open-sourced, or will it be open-sourced?

**Comment by Haoye17** [comment]
Hi @RobitsG,

The data in the RLAIF-V dataset contains preference pairs for **every iteration** in our training, including the training for LLaVA 1.5 7B and OmniLMM 12B.

The code for data generation has been open-sourced, you can follow the instructions in [Data Generation](https://github.com/RLHF-V/RLAIF-V/blob/main/README.md#data-generation) for more details.
